src/cli.js

```diff
@@ -38,6 +38,7 @@ function parse(argv) {
 	const config = {
 		help: removeFromList(argv, "--help") || removeFromList(argv, "-h"),
 		everything: !(removeFromList(argv, "--errors-only")),
+		offline: removeFromList(argv, "--offline"),
```

Comment:
This option MUST be included in the [`--help` message](https://github.com/ericcornelissen/depreman/blob/0ba1f6d2efaa7990b4ec9ef11dec4266a0f849df/src/main.js#L32-L46).